### PR TITLE
completes prep-javascript-variables assignment

### PR DESCRIPTION
**Pull request by aklepper20**
prep-javascript-variables || Aly Klepper

image

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    /* your code here */
+    var fullName = "Aly Klepper";
+    var numberOfPets = 1;
+    var likesVegetables = true;
+    var nothing = null;
+    var never =
```
Comment (thebearingedge) [inline prep-javascript-variables/index.html:16]: For this variable, you can leave out the `=` entirely.

```js
var never;
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice! Just one small change.

File: prep-javascript-variables/index.html (hunk 1 above)
Comment (aklepper20) [reply prep-javascript-variables/index.html:16]: image
fixed, updated and pushed up to git

**Review by thebearingedge — APPROVED** [review]
Approved!

**Pull request merged**